resources/popup.html

```diff
@@ -39,7 +40,7 @@
                 </a>
             </div>
             <div class="col s6">
-                <a class="btn waves-effect lime waves-light tooltipped" href="../analytics/index.html" target="_blank"
+                <a id="analytics-link" class="btn waves-effect lime waves-light tooltipped" target="_blank"
```

Comment:
Why make this harder than it is? If the only class that uses this target why not place it in the popup.